api/app/controllers/spree/api/customer_returns_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Spree
+  module Api
+    class CustomerReturnsController < Spree::Api::BaseController
```

Comment:
In think should be better inherit from `ResourceController`

Reply:
I checked and `Spree::Api::ResourceController` is only used in `Spree::Api::UsersController`. Maybe it was the wrong abstraction and it's better to deprecate it at this point (in another PR)?